net/speedtest/speedtest_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package speedtest
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestDownload(t *testing.T) {
+	// start up the speedtest server with a hardcoded port and address
```

Comment:
It's not hardcoded, below. So either the code or the comment is wrong. I'm guessing the comment.